src/include/kernel/abi.h

```diff
@@ -30,7 +30,7 @@
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
 #define SOF_ABI_MINOR 24
-#define SOF_ABI_PATCH 0
+#define SOF_ABI_PATCH 1
```

Comment:
If we need to change structs shared with IPC3, this needs a MINOR bump and be tagged with "ABI".